### PR TITLE
added github issue templates

### PR DESCRIPTION
**Pull request by panoskyriakis**
Created issue templates for the following:

1. Bug report
2. Feature Request
3. Documentation
4. Question

Correct render not verified; must merge to main.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Contact Us
+    url: team@nevron.ai
```
Comment (stripeli) [inline .github/ISSUE_TEMPLATE/config.yaml:4]: Why this email? Have we created an alias?

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+name: 🚀 Feature request
+description: Submit a proposal or request for a new feature in prompttools!
```
Comment (stripeli) [inline .github/ISSUE_TEMPLATE/feature_request.yaml]: What is a prompt tool? Be more specific.

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+labels: ["enhancement"]
+
+body:
+  - type: textarea
+    attributes:
+      label: 🚀 The feature
+      description: >
+        A clear and concise description of the feature proposal
```
Comment (stripeli) [inline .github/ISSUE_TEMPLATE/feature_request.yaml]: Put period at the end of the sentence.

File: .github/ISSUE_TEMPLATE/question.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+name: 🙋 Question and Discussion
+description: Discuss or ask question about how to use prompttools and what the best practice may be.
```
Comment (stripeli) [inline .github/ISSUE_TEMPLATE/question.yaml]: Type: ask questions on how to ...
Similar to the comment earlier. Which prompt tool are we referring to?

**Review by stripeli — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yaml (hunk 1 above)
Comment (panoskyriakis) [reply .github/ISSUE_TEMPLATE/config.yaml:4]: Yes

**Review by dstripelis — APPROVED** [review]

**Review by stripeli — APPROVED** [review]

**Pull request merged**